File: code/src/test/java/cernx/colt/matrix/tlong/impl/DenseLargeLongMatrix2DTest.java
package cernx.colt.matrix.tlong.impl;

import cernx.colt.matrix.tlong.LongMatrix2DTest;

public class DenseLargeLongMatrix2DTest extends LongMatrix2DTest {

    public DenseLargeLongMatrix2DTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseLargeLongMatrix2D(NROWS, NCOLUMNS);
        B = new DenseLargeLongMatrix2D(NROWS, NCOLUMNS);
        Bt = new DenseLargeLongMatrix2D(NCOLUMNS, NROWS);
    }
}

File: code/src/test/scala/cern/colt/matrix/tlong/impl/DenseLargeLongMatrix2DTest.scala
package cern.colt.matrix.tlong.impl

import cern.colt.matrix.tlong.LongMatrix2DTest
//remove if not needed
import scala.collection.JavaConversions._

class DenseLargeLongMatrix2DTest(arg0: String) extends LongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new DenseLargeLongMatrix2D(NROWS, NCOLUMNS)
    B = new DenseLargeLongMatrix2D(NROWS, NCOLUMNS)
    Bt = new DenseLargeLongMatrix2D(NCOLUMNS, NROWS)
  }
}
